### PR TITLE
Fix wrong link

### PR DESCRIPTION
**Pull request by xichengliudui**
Fix wrong link

### CODE REVIEW

**Comment by nikhita** [comment]
https://kubernetes.io/docs/setup/pick-right-solution/ has a link to the getting started guide here, so maybe we shouldn't really remove this...

A link fix should be fine for now. :+1:

File: contributors/devel/writing-a-getting-started-guide.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ A guide refers to scripts, Configuration Management files, and/or binary assets
 the combination of all these things needed to run on a particular type of infrastructure a
 **distro**.
 
-[The Matrix](../../docs/getting-started-guides/README.md) lists the distros.  If there is already a guide
+[The Matrix](`../../docs/getting-started-guides/README.md`) lists the distros.  If there is already a guide
```
[the remark this answers is not in the document]
Comment (xichengliudui) [reply contributors/devel/writing-a-getting-started-guide.md]: OK,Thanks

**Comment by nikhita** [comment]
@xichengliudui Thanks, the change looks good!

Can you also fix the commit message and the PR title? Right now it says "Disable invalid links".

**Comment by xichengliudui** [comment]
@nikhita Thanks,I have modified it.

**Pull request merged**